package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "@signoz-icons/icons",
+  "version": "1.0.0",
```

Comment:
shouldn't we start with `0.0.0` version and then keep the version number increments on top of it? `1.0.0` is generally used when the project is marked as stable and has achieved a decent good state.

Reply:
@YounixM : yes .. We can start with minor / patch version update till we add all icons and then do a `1.0.0`

Reply:
@rahulkeswani101 : Please take care of this.

Reply:
https://semver.org/#spec-item-4